README.md

```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
 an identity provider (IdP) and a service provider (SP).
 
 This charm simplifies configuration of SAML SPs by providing a single point
-of configuration for all the requirers using the same SAML entity. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
+of configuration for all the requirers using the same SAML entity.
```

Comment:
Nitpick: Personally I think this paragraph was fine as it was before, but I don't have a strong opinion about it.

Reply:
I changed it to be better aligned with https://github.com/canonical/is-charms-template-repo and the changes in the Indico operator README. Let me know if you'd prefer to keep it as it was